### ISSUE TITLE
TypeScript error on ToastContainer

### ISSUE THREAD

**Issue by JoshRandall-Andculture** [fkhadra/react-toastify#176]
When we add the `<ToastContainer />` element in a .tsx file, we receive the error `JSX element type 'ToastContainer' does not have any construct or call signatures`

If we change the definition for the ToastContainer class in `index.d.ts` to include the `any` type for the component state it no longer throws that error.  Can this change be made in the repo and re-published to npm?

Doesn't work (current definition):
```javascript
export class ToastContainer extends React.Component<ToastContainerProps> {}
```

Does work (manually modified locally):
```javascript
export class ToastContainer extends React.Component<ToastContainerProps, any> {}
```

**Comment by fkhadra** [comment]
@JoshRandall-Andculture  thanks for the feedback i'll do it now

**Comment by JoshRandall-Andculture** [comment]
Awesome, thanks for the quick response @fkhadra!

**Comment by fkhadra** [comment]
@JoshRandall-Andculture I published the v4.0.1. I had to fix a build issue in the mean time.

**Comment by fkhadra** [comment]
I close the issue, feel free to tell if something else wrong.
